ui/static/js/app.js

```diff
@@ -618,3 +618,23 @@ function showToast(label, iconElement) {
 function goToAddSubscription() {
     window.location.href = document.body.dataset.addSubscriptionUrl;
 }
+
+/**
+ * save player position to allow to resume playback later
+ * @param {Element} playerElement
+ */
+function handlePlayerProgressionSave(playerElement) {
+    const currentPositionInSeconds = Math.ceil(playerElement.currentTime); // we do not need a precise value
```

Comment:
I'd prefer `floor`. I'd rather hear the same partial second over again than miss content.

Reply:
Seems fair. I'll change that. :+1:  :thanks.